source/Halibut.TestUtils.CompatBinary.Base/SettingsHelper.cs

```diff
@@ -83,5 +83,17 @@ static ServiceConnectionType AsServiceConnectionType(string s)
 
             throw new Exception($"Unknown service type '{s}'");
         }
+
+        public static string CompatBinaryStayAliveLockFile()
+        {
+            var tentacleCertPath = GetSetting("CompatBinaryStayAliveFilePath");
+            if (tentacleCertPath == null || tentacleCertPath.Length == 0)
+            {
+                throw new Exception("Env var CompatBinaryStayAliveFilePath must be set");
+            }
+            
+            Console.WriteLine($"Will die when the following file can be locked or is deleted '{tentacleCertPath}'.");
```

Comment:
Nit: I feel this line doesn't belong to the method that gets the file name. But rather, to the place where we use this file name (i.e. in `StayAliveUntilHelper`)